docs/source/contributing/developing.adoc

```diff
@@ -131,10 +131,9 @@ If you seeing Glide errors during `make vendor`, you can clear the Glide cache v
 == Building Minishift
 
 [[build-minishift-binary]]
-=== Building the Minishift binary
+=== Building the Minishift Binary
```

Comment:
Articles are usually written as lowercase; 'a', 'an', and 'the'.

The point here would be, consistency. In the documentation there is
  * Building the Minishift binary
  * Running the Minishift binary